samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClientCurrent.ts

```diff
@@ -264,11 +267,17 @@ const networkRequestViaHttps = <T>(
     const isPostRequest = httpMethod === HttpMethod.POST;
     const body: string = options?.body || "";
 
+    const url = new URL(urlString);
     const emptyHeaders: Record<string, string> = {};
     const customOptions: https.RequestOptions = {
+        hostname: url.hostname,
```

Comment:
I think you should use `host` instead of `hostname`, because `host` includes the port - see https://stackoverflow.com/questions/17184791/node-js-url-parse-and-pathname-property

Reply:
What do you mean by "We should not be writing this type of code"? What "type of code" are you referring to? What am I bound to get wrong? I'm using the URL as it's defined on the MDN web docs [here](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL). I'm following the example RequestOptions as defined in the NodeJS docs [here](https://nodejs.org/api/https.html#httpsrequesturl-options-callback), and it does appear that I forgot the port.

It looks like:
```
const options = {
  hostname: 'encrypted.google.com',
  port: 443,
  path: '/',
  method: 'GET',
};
```

Reply:
Actually, because this is an htpps request, the port is automatically assumed to be 443.